### PR TITLE
🛀 Remove topups part 2

### PR DESCRIPTION
**Pull request by rowanseymour**
* Removes creating, updating and managing topups
 * Removes the transfer credits view

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -485,7 +484,7 @@
         "whatsapp_cloud_connect",
         "workspace",
     ),
-    "orgs.topup": ("manage",),
```
Comment (norkans7) [inline temba/settings_common.py:488]: `orgs.topup` line should be removed here entirely

**Review by norkans7 — CHANGES_REQUESTED** [review]
just the 2 changes

**Review by susanm74 — APPROVED** [review]
LGTM 👍

File: temba/settings_common.py (hunk 1 above)
Comment (rowanseymour) [reply temba/settings_common.py:488]: Ah I was thinking that would remove the read/list permissions for the views I'm leaving for now

**Pull request merged**